src/model/request/DeployEthErc20.ts

```diff
@@ -11,39 +11,7 @@ import {
   ValidateNested,
 } from 'class-validator';
 import {Fee} from './Fee';
-import { PrivateKeyOrSignatureId } from './PrivateKeyOrSignatureId'
+import { DeployErc20 } from './DeployErc20'
 
-export class DeployEthErc20 extends PrivateKeyOrSignatureId {
-
-    @IsNotEmpty()
-    @Length(1, 100)
-    @Matches(/^[a-zA-Z0-9_]+$/)
-    public name: string;
-
-    @IsNotEmpty()
-    @Length(1, 30)
-    public symbol: string;
-
-    @IsNotEmpty()
-    @Length(42, 42)
-    public address: string;
-
-    @IsNotEmpty()
-    @IsNumberString()
-    @MaxLength(38)
-    @Matches(/^[+]?((\d+(\.\d*)?)|(\.\d+))$/)
-    public supply: string;
-
-    @Min(1)
-    @Max(30)
-    public digits: number;
-
-    @IsOptional()
-    @Min(0)
-    public nonce?: number;
-
-    @IsOptional()
-    @ValidateNested()
-    @Type(() => Fee)
-    public fee?: Fee;
+export class DeployEthErc20 extends DeployErc20 {
```

Comment:
this class is not needed. we can use Base class for ETH

Reply:
ok